src/metadata/copy_var/config.vsh.yaml

```diff
@@ -0,0 +1,68 @@
+name: copy_var
```

Comment:
When reading `copy`, I immediately assumed that you meant to copy from one mudata object to another. I would prefer something different here:

```suggestion
name: duplicate_var
```